README.md

```diff
@@ -19,7 +19,7 @@ It's possible to install the Data Dictionary within a Python environment.
 
 ```sh
 git clone git@github.com:iterorganization/IMAS-Data-Dictionary.git
```

Comment:
This is fine and I will approved but the alternative way of fixing this (which may have handled changes of upper/lowercase on the GitHub URL side) would have been to do:
```suggestion
git clone git@github.com:iterorganization/IMAS-Data-Dictionary.git imas-data-dictionary
```
and to clone into a named directory.